source/blog/2016-06-14-rspec-3-5-has-been-released.md

```diff
@@ -0,0 +1,345 @@
+---
+title: RSpec 3.5 has been released!
+author: Sam Phippen
```

Comment:
Since I wrote up a bunch of the notable changes section, should we change this to `Sam Phippen and Myron Marston`?